state/syncer/validatorAccountsSyncer.go

```diff
@@ -77,10 +77,7 @@ func (v *validatorAccountsSyncer) SyncAccounts(rootHash []byte) error {
 		return err
 	}
 
-	err = mainTrie.GetStorageManager().Put([]byte(common.TrieSyncedKey), []byte(common.TrieSyncedVal))
-	if err != nil {
-		log.Warn("error while putting trieSynced value into main storer after sync", "error", err)
-	}
+	markStorerAsSyncedAndActive(mainTrie)
```

Comment:
can we move this function in the trie? So we can have: `mainTrie.MarkStorerAsSyncedAndActive()`